bHY6yh-de3258/cod-bxxqqq_bHY6yh-de3258.xml

```diff
@@ -0,0 +1,236 @@
+              <lb ed="#B"/>declinatur a malo. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e264">
+      ¶ <ref xml:id="bHY6yh-de3258-Rd1e287">24 capitulo</ref>
+        <quote xml:id="bHY6yh-de3258-Qd1e266">Time 
+        <lb ed="#B"/>dominum fili mi et regem</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e272">
+      ¶ <ref xml:id="bHY6yh-de3258-Rd1e297">Idem in ecclesiastico
+      <lb ed="#B"/>capitulo 7</ref> <quote xml:id="bHY6yh-de3258-Qd1e274">Qui timet dominum nihil ne<lb ed="#B" break="no"/>gligit</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e280">
+              ¶ <ref xml:id="bHY6yh-de3258-Rd1e310">capitulo 12 et fi</ref> <quote xml:id="bHY6yh-de3258-Qd1e284">Deum time et 
+              <lb ed="#B"/>mandatum eius obserua hoc est 
+              <lb ed="#B"/>enim omnis homo. </quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e290">
+               <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de3258-Rd1e322">Tobie hystoria <!--line jumps up-->capitulo 4</ref> <quote xml:id="bHY6yh-de3258-Qd1e294">multa bo<!--line jumps down-->na habebimus si ti
+                <lb ed="#B"/>muerimus deum: et recesserimus ab omni 
+                <lb ed="#B"/>peccato et fecerimus bene.</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e303">
+      ¶ <ref xml:id="bHY6yh-de3258-Rd1e335">Socrates in exhorta<!--line jumps up-->tionibus <!--line jumps down-->capitulo II circa 
+      <lb ed="#B"/>princium</ref> <quote xml:id="bHY6yh-de3258-Qd1e305">Deum quidem verere</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e311">
+              <lb ed="#B"/>
+                <ref xml:id="bHY6yh-de3258-Rd1e348">Iesus filius sirach in ecclesio capitulo 1</ref> <quote xml:id="bHY6yh-de3258-Qd1e315">Timor 
+                <lb ed="#B"/>domini gloria et gloratio letitia et 
+                <lb ed="#B"/>corona exultationis ¶ Timor 
+        <lb ed="#B"/>domini delectabit cor et dabit letitiam 
+        <lb ed="#B"/>et gaudium et longitudinem dierum</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e325">
+      ¶ <quote xml:id="bHY6yh-de3258-Qd1e327"> Timere dominum
+        <lb ed="#B"/>bene erit in extremis: et in die defunctionis suc
+        <lb ed="#B"/>benedicetur.</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e335">
+      ¶ <quote xml:id="bHY6yh-de3258-Qd1e337">Timor domini conscientiae 
+              <lb ed="#B"/>religiositas custodiet et iustificabit 
+              <lb ed="#B"/>cor</quote>
+            </p>
+            <p xml:id="bHY6yh-de3258-d1e343"> /
+      ¶<quote xml:id="bHY6yh-de3258-Qd1e345"> Timenti deum bene erit et in die<lb ed="#B" break="no"/>bus
+        consummationis illius benedictur</quote>
+            </p>
+                  <pb ed="#B" n="7-r"/>
+                  <cb ed="#B" n="a"/>
+            <p xml:id="bHY6yh-de3258-d1e359"> /
```

Comment:
These slashes look like they are left over from the venice transcription (Let me know if I'm wrong. Otherwise go ahead and remove them throughout)